src/functionaltest/resources/functional-test.properties

```diff
@@ -0,0 +1,2 @@
+test.selenium.firefox.BZip2File.linux = https://download.mozilla.org/?product=firefox-61.0.1-ssl&os=linux64&lang=en-GB
```

Comment:
This is more proper url to use which has no arguments that can cause address lookup issues:
https://ftp.mozilla.org/pub/firefox/releases/61.0.1/linux-x86_64/en-US/firefox-61.0.1.tar.bz2
With this you can extract versions and fileName from the address.
Remove the "......linux.name" and use only url variable.
And add url to the property settings parameter, e.g: "test.selenium.firefox.BZip2File.url.linux "

Reply:
Done